models/repmaint.json

```diff
@@ -1,7 +1,8 @@
 {
     "fact_table": "repmaint_facts",
     "label": "Repairs and Maintenance",
-    "description": "This is based on the RME return form.",
+    "description": "Money spent on repairs and maintenance of assets.",
+    "return_forms": ["RME"],
```

Comment:
I think I asked whether there's value in showing it in a dedicated cube. I think they said the value in this is the breakdown in this view while capex only shows the total